conda-recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
   - pyyaml >=5.4.0
   - requests >=2.26.0
   - simplejson >=3.17.0
+  - pykerberos
```

Comment:
small nitpick: the packages in meta.yaml and requirements.txt are in alphabetical order aside from this addition